docs/csharp/getting-started/interactive-lessons/hello-world.md

```diff
@@ -0,0 +1,274 @@
+---
+title: Hello World - C# Interactive Lessons | Microsoft Docs
+description: Write your first C# program
+keywords: C#, Get Started, Hello World, interactive, lesson
+author: billwagner
+ms.author: wiwagn
+ms.date: 06/02/2017
+ms.topic: article
+ms.prod: .net
+ms.technology: devlang-csharp
+ms.devlang: csharp
+---
+
+# Hello world lesson
+
+You'll create your first C# program in this lesson. You'll learn the
+structure of a C# program and the common elements that are part of every
+C# program. You'll also learn the basic tools to manipulate text using
+the C# programming language.
+
+## Prerequisites
+
+This lesson has no prerequisites.
+
+## Hello world
+
+Enter the following code into the code editor:
+
+```csharp
+using System;
+
+namespace HelloWorld
+{
+    public class Program
+    {
+        public static void Main()
+        {
+            Console.WriteLine("Hello World!");
+        }
+    }
+}
+```
+
+Run this program, and you'll see the text `Hello World!`printed. Congratulations,
+you've written you're first C# program.
+
+## Understanding what you've created
+
+Quite a few of the elements you'll use in every C# program are in this first
+sample. These elements form the structure of C# programs. Let's examine them
+before you move on to modifying this program. Rather than working line-by-line,
+let's examine the building blocks from the inside out. This program, like all
+programs, is made up of *statements*, *methods*, *classes*, and *namespaces*.
```

Comment:
:+1:, this is statement:

> This program, like all programs, is made up of *statements*, *methods*, *classes*, and *namespaces*.

Is only true for C#.  Other languages don't have have statements.  We should clarify that this is for "all _C#_ programs".